<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryForm</class>
 <widget class="QWidget" name="CategoryForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Categories</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="leftVerticalLayout">
     <item>
      <widget class="QGroupBox" name="typeGroupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="incomeRadioButton">
          <property name="text">
           <string>Income</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="expenseRadioButton">
          <property name="text">
           <string>Expense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="incomeAndExpenseRadioButton">
          <property name="text">
           <string>Income and Expense</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="controlsHorizontalLayout">
       <item>
        <widget class="QToolButton" name="expandAllToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="collapseAllToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="editToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLineEdit">
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
